src/main/java/org/benetech/servicenet/service/annotation/ConfidentialFilterAspect.java

```diff
@@ -0,0 +1,36 @@
+package org.benetech.servicenet.service.annotation;
+
+import org.apache.commons.lang3.BooleanUtils;
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Aspect;
+import org.benetech.servicenet.domain.AbstractEntity;
+import org.springframework.stereotype.Component;
+
+@Aspect
+@Component
+public class ConfidentialFilterAspect {
```

Comment:
We should add java doc for this annotation because it might be confusing when some will start to search it online